sqlmesh/core/context.py

```diff
@@ -842,6 +842,13 @@ def migrate(self) -> None:
         """
         self._new_state_sync().migrate()
 
+    def rollback(self, schema_version: int = 0) -> None:
```

Comment:
maybe get rid of all the default values here

Reply:
this is still here

Reply:
Thanks, I forgot to add the file in git before committing.